lib/matplotlib/artist.py

```diff
@@ -903,6 +903,27 @@ def matchfunc(x):
             artists.append(self)
         return artists
 
+    def get_zdata(self, event):
+        """
+        Get the zdata for a given event, as a string message
+        """
+        return ''
```

Comment:
This allows the `Artist` to control the display of numbers, similar to the existing `ax.format_coord`.